src/cavendish_particle_tracks/_main_widget.py

```diff
@@ -365,7 +384,9 @@ def _on_click_radius(self) -> None:
                 QTableWidgetItem(str(self.data[selected_row].radius_cm)),
             )
 
-            print("Modified particle ", selected_row)
+            napari.utils.notifications.show_info(
+                "Radius added to particle " + str(selected_row)
+            )
```

Comment:
I added this at the end, to show some feedback to the user that the measurement has been recorded. The notifications seem to stay though, as in they do not go away after some time.

Do you think it is annoying?

Reply:
It actually does go away.... it's just a bit slow.